pkg/tests/api/dashboards/api_dashboards_test.go

```diff
@@ -163,6 +164,9 @@ providers:
 		Login:          "admin",
 	})
 
+	// give provisioner some time to load dashboards
+	time.Sleep(1 * time.Second)
```

Comment:
I suggest we try to find the appropriate fix for the data race. Sleeping is never a fix for data races, that only makes them harder to debug when they happen at the cost of also making tests unnecessarily slower to run.